packages/ngtools/webpack/src/transformers/ctor-parameters.ts

```diff
@@ -175,6 +177,20 @@ function typeReferenceToExpression(
       return ts.createIdentifier('Number');
     case ts.SyntaxKind.TypeReference:
       const typeRef = node as ts.TypeReferenceNode;
+      debugger;
```

Comment:
oops!!!

Reply:
We should really have a lint check for that.

Reply:
Yeah, same for console.log